nUpdate.Administration/UI/Dialogs/ProjectEditDialog.cs

```diff
@@ -994,9 +994,13 @@ private void continueButton_Click(object sender, EventArgs e)
                     _useStatistics = useStatisticsServerRadioButton.Checked;
                 }));
 
-                bool synchronizeData = Popup.ShowPopup(this, SystemIcons.Question, "Automatically synchronize data?",
-                        "nUpdate Administration may encounter differences between your remote configuration and the local changes that you've made. Should nUpdate Administration try to synchronize these changes or do you want to keep them locally? Choose the latter option, if you just fixed a problem with the data (e.g. a typo due to which nUpdate Administration could not connect to the server).",
-                        PopupButtons.YesNo) == DialogResult.Yes;
+                bool synchronizeData = (bool)Invoke(
```

Comment:
Same thing with the commit message here (see the copyright symbol commit).

Thanks for fixing this. Seems that I've not seen that this needs to be invoked. Maybe because I've added this code much later.